javascript/map.js

```diff
@@ -0,0 +1,51 @@
+const map = new maplibregl.Map({
+  container: 'map',
+  style: 'https://demotiles.maplibre.org/style.json', // stylesheet location
```

Comment:
I'd like to suggest this change so that we don't load the demo tiles anymore:
```suggestion
  style: {
    version: 8,
    sources: {
      'basemap-satellite': {
        type: 'raster',
        tiles: ['https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}'],
        tileSize: 256
      },
      'basemap-light': {
        type: 'raster',
        tiles: ['https://a.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}{r}.png'],
        tileSize: 256
      },
    },
    layers: [],
  },
```

Reply:
We need some glyphs to be able to display labels and so. This is not the correct format for the style. But I will look at it

Reply:
Let me know if you need some help with this. Thanks!